### ISSUE TITLE
CZ-TAW1 dead after two days of power off

### ISSUE THREAD

**Issue by stefanbbb** [lsochanowski/GoHeishaMon#10]
Hi, 
The CZ-TAW1 was running like a charme directly after installation, but after two days of shutting down the power, it will not restart. Just the green power LED and nothing else. Sometimes also the white one, but not the third, and no connection at all. No reaction to HW-Reset (3buttons simultanously)
Who can help?

**Comment by lsochanowski** [comment]
Did you try ssh into taw ? It respond to Ping ? Or smth ?

**Comment by stefanbbb** [comment]
ssh not possible, because mqtt is dead. Ping is possible

**Comment by lsochanowski** [comment]
Ssh have Nothing in common with mqtt but if ssh is unavible (thru putty or smth ) did you try to run luci ? It’s working ? When There is No ssh propably somehow you return to orginal firmware and aquarea cloud

**Comment by stefanbbb** [comment]
I have no experience with luci, but how can I get back to original SW?

**Comment by stefanbbb** [comment]
currently original SW is not running, because I have no blue middle LED and no status link. System is just sticking with the green power LED. But there is some request to USB-Stick during startup.

**Comment by lsochanowski** [comment]
Ok that’s good you have certainty ssh is not runing ? Did you try putty ?

**Comment by stefanbbb** [comment]
yes

**Comment by stefanbbb** [comment]
I tried on port 22, but no reaction

**Comment by lsochanowski** [comment]
Ok first did you try to reboot taw  with power without USB drive inserted ? What files did you have on USB drive ?

**Comment by stefanbbb** [comment]
I tried with and without USB-Drive. On the drive are just the three files you requested in your installation advice (GoHeishaMonConfig.new and the two bins mentioned by you)

**Comment by lsochanowski** [comment]
Ok can you paste Here config.new ( without passwords etc )

**Comment by stefanbbb** [comment]
It's named"GoHeishaMonConfig.new" and here ist he content:

Readonly=true
Device="/dev/ttyUSB0"
Loghex=true
ReadInterval=1
Aquarea2mqttCompatible=true
Aquarea2mqttPumpID="B07-------"
Mqtt_topic_base="panasonic_heat_pump/sdc"
Mqtt_set_base="panasonic_heat_pump"
ForceRefreshTime=60
MqttServer="192.168.86.146"
MqttPort="1884"
MqttLogin="iobroker"
MqttPass="********"
SleepAfterCommand=3
MqttClientID="Panasonic_Klima"
MqttKeepalive=60

**Comment by lsochanowski** [comment]
which version of heishamon did you use ?

**Comment by stefanbbb** [comment]
1.0.1.35

**Comment by MiG-41** [comment]
This USB request is that GoHeishaMon is checking ,if on USB stick there is a new config file GoHeishaMonConfig.new , and if there is , it copy it and use it . 
So can you change for test:
Aquarea2mqttCompatible=false 
and remove ID from Aquarea2mqttPumpID ? ( Pleae also delete it from post above , this ID is not a thing with shuld be public).
and add: 
EnableCommand=true

Please also restart mqtt server ,and check once again that mqtt server havn't changed ip , and all login , password and port are still valid.

**Comment by stefanbbb** [comment]
Done, but no change.  I removed the ID like this:
Aquarea2mqttPumpID=""
I that what you mean?

**Comment by MiG-41** [comment]
Yes...
No change  means that you everytime reset cz-taw1  by unplugging The Power?

**Comment by stefanbbb** [comment]
Yes, exactly

**Comment by stefanbbb** [comment]
Small change:
with usb drive in, I get the white LED after some time during booting, without usb drive not.
But still no connection to ssh or mqtt

**Comment by MiG-41** [comment]
So when The White LED apear try to Press  3 buttons to go back to orginal. 
But i believe There od stilo something wrong with mqtt ,either your settings ,or serwer stoped to let The GoHeishamon.
To have ssh and/or luci you need to have connection to mqtt first nad Then Send values, like mentioned in Read.me.
You can try to start A new MQTT server on other machine?

**Comment by stefanbbb** [comment]
This is really confusing. System is not reacting by pressing the 3 buttons. Sometimes I have the white LED after rebooting, sometimes not. 
iobroker got rebootet and I installed a new instance of mqtt server on port 1885 (changed the config file accordingly)
 But still no reaction of the system. Ping is there, but no mqtt and no ssh.

**Comment by MiG-41** [comment]
SSH you will not have untill you will perform sending commends via mqtt....( like in Readme,md)
Are you sure that new config has applied ? Do you see a blinkin LED on pendrive ?

If you are sure ,that confg , and mqtt server is ok , then only way to have the acces to cz-taw1 now is  to connect usb ttl converter (with voltage level 3,3V) to CZ-TAW1 to this pins: https://github.com/Egyras/HeishaMon/issues/1#issuecomment-558364184
RX of cz-taw1 to TX of converter ,and TX of cz-taw1 to RX of converter , GND to GND .
Only this 3 lines , no more!

![20200611_181700](https://user-images.githubusercontent.com/57936100/84413307-b71ff300-ac10-11ea-8b4c-1d032ad5349e.jpg)

**Comment by stefanbbb** [comment]
ok, and when I have access, what should I do?

**Comment by MiG-41** [comment]
Then i.e. via Putty and COM connection ( on Windows offcourse) you will have similar acces like ssh.
and if you will show us what is the couse of the problem you could put commands:
ps - to see if GoHeishaMon procces is runing, if not , then first show me via PM a result of command:
 uci show ( this i need to se pure configuration ,without performing firstboot yet).
and then perform a firstboot typing command : firstboot ,and then conforming it by pressing y ,and make a reboot via command: reboot.
In this way you could connect via ssh ( hopfully) , and check once again via ps if proces exist , if not ,run it manually via command: /usr/bin/GoHeishaMon_MIPSUPX 
( you couldn't do it via COM ,becouse GoHeishamon for sum unknown reason change the port speed and connection via COM couses ,that logs vill not apear anymore).
In this way we will see logs of GoHeishaMon ,and can tell more what have happen.

You can also do a comannd:  fwupdate sw , and then perform a reboot  , and you will have SmarCloud again....

**Comment by stefanbbb** [comment]
OK, first of all thank you very much for the great support so far from you both. I will try this advice this night, but it seems to be a greater job.  
Maybe the quickest solution ist to go back to original SW and start the initial installation again, what do you think?

**Comment by MiG-41** [comment]
That will be the quickest , but in this way we will not find out what went wrong.

**Comment by lsochanowski** [comment]
please try one more think , after editing config on usb drive try to reboot taw with that drive and wait around 6-8min. with this pendrive inserted if pendrive have some led please watch this led .. every 5 minutes goheisha will try to update config maybe this will work

**Comment by MiG-41** [comment]
And after copy of config perform reboot as well ,since config will e not applied automatically....

**Comment by stefanbbb** [comment]
Ok, I got it.
Serial connection is wired. But in putty i get no result.
In Eclipse I get some chryptic characters.
I've tried with different baudrates, but no result.
Which parameters do I need for the com port?

**Comment by stefanbbb** [comment]
![image](https://user-images.githubusercontent.com/66436566/84577593-0f482800-adbe-11ea-803d-42263d1e85b0.png)
Here the result in terminal window

**Comment by MiG-41** [comment]
115200baud  ,like mentioned in link .
Other parameters: 8 data bits , 1 stop bit , none parity, XON/XOFF flow control.
In link you have also a zip file with text , how cz-taw1 starts. ( should be identical).

**Comment by MiG-41** [comment]
I forgot ,that when GoHeishaMon starts , it will block console....
Please start putty , and then power on cz-taw1. When you see the line  
"Press the [f] key and hit [enter] to enter failsafe mode"
Do it what it says. ( it will stop booting , and GoHeishaMon will not starts , that gives you a posibility to enable ssh via firstboot).

**Comment by stefanbbb** [comment]
No, there is no real data in putty, just some creepy characters directly after power up the CZ-TAW1.
Seems to be, I've no luck with this device. Do you have any other ideas? Otherwise I have to switch to some ESP-8266 Solution. A new CZ-TAW1 is too expensive

**Comment by stefanbbb** [comment]
Here the screenshot from putty:
![image](https://user-images.githubusercontent.com/66436566/84594627-e96d6280-ae53-11ea-8e14-5394493e6f78.png)

**Comment by stefanbbb** [comment]
Great, I tried another serial interface, and now it works!!
Here the output:


U-Boot 1.1.4-g4d980152-dirty (Jun 24 2016 - 10:12:23)

cus531 - Honey Bee 2.0DRAM:
sri
Honey Bee 2.0
ath_ddr_initial_config(195): (16bit) ddr2 init
tap = 0x00000003
Tap (low, high) = (0x9, 0x21)
Tap values = (0x15, 0x15, 0x15, 0x15)
setup gpio
64 MB
Top of RAM usable for U-Boot at: 84000000
Reserving 199k for U-Boot at: 83fcc000
Reserving 192k for malloc() at: 83f9c000
Reserving 44 Bytes for Board Info at: 83f9bfd4
Reserving 36 Bytes for Global Data at: 83f9bfb0
Reserving 128k for boot params() at: 83f7bfb0
Stack Pointer at: 83f7bf98
Now running in RAM - U-Boot at: 83fcc000
Flash Manuf Id 0xc8, DeviceId0 0x40, DeviceId1 0x18
flash size 16MB, sector count = 256
Flash: 16 MB
*** Warning *** : PCIe WLAN Module not found !!!
In:    serial
Out:   serial
Err:   serial
Net:   ath_gmac_enet_initialize...
No valid address in Flash. Using fixed address
No valid address in Flash. Using fixed address
ath_gmac_enet_initialize: reset mask:c02200
Scorpion ---->S27 PHY*
S27 reg init
: cfg1 0x800c0000 cfg2 0x7114
eth0: 00:03:7f:09:0b:ad
athrs27_phy_setup ATHR_PHY_CONTROL 4 :1000
athrs27_phy_setup ATHR_PHY_SPEC_STAUS 4 :10
eth0 up
Honey Bee ---->  MAC 1 S27 PHY *
S27 reg init
ATHRS27: resetting s27
ATHRS27: s27 reset done
: cfg1 0x800c0000 cfg2 0x7214
eth1: 00:03:7f:09:0b:ad
athrs27_phy_setup ATHR_PHY_CONTROL 0 :1000
athrs27_phy_setup ATHR_PHY_SPEC_STAUS 0 :10
athrs27_phy_setup ATHR_PHY_CONTROL 1 :1000
athrs27_phy_setup ATHR_PHY_SPEC_STAUS 1 :10
athrs27_phy_setup ATHR_PHY_CONTROL 2 :1000
athrs27_phy_setup ATHR_PHY_SPEC_STAUS 2 :10
athrs27_phy_setup ATHR_PHY_CONTROL 3 :1000
athrs27_phy_setup ATHR_PHY_SPEC_STAUS 3 :10
eth1 up
eth0, eth1
Setting 0x181162c0 to 0x4081a100
Hit any key to stop autoboot:  0
## Booting image at 9fe80000 ...
   Image Name:   MIPS OpenWrt Linux-3.14
   Created:      2020-03-10   7:21:32 UTC
   Image Type:   MIPS Linux Kernel Image (lzma compressed)
   Data Size:    1222926 Bytes =  1.2 MB
   Load Address: 80060000
   Entry Point:  80060000
   Verifying Checksum at 0x9fe80040 ...OK
   Uncompressing Kernel Image ... OK
No initrd
## Transferring control to Linux (at address 80060000) ...
## Giving linux memsize in bytes, 67108864

Starting kernel ...

[    0.000000] Linux version 3.14.0 (root@A2W-LINUX) (gcc version 4.8.3 (OpenWrt/Linaro GCC 4.8-2014.01 r40838) ) #1 Tue Mar 10 07:21:14 UTC 2020
[    0.000000] bootconsole [early0] enabled
[    0.000000] CPU0 revision is: 00019374 (MIPS 24Kc)
[    0.000000] SoC: Qualcomm Atheros QCA9533 rev 2
[    0.000000] Determined physical RAM map:
[    0.000000]  memory: 04000000 @ 00000000 (usable)
[    0.000000] User-defined physical RAM map:
[    0.000000]  memory: 04000000 @ 00000000 (usable)
[    0.000000] Initrd not found or empty - disabling initrd
[    0.000000] Zone ranges:
[    0.000000]   Normal   [mem 0x00000000-0x03ffffff]
[    0.000000] Movable zone start for each node
[    0.000000] Early memory node ranges
[    0.000000]   node   0: [mem 0x00000000-0x03ffffff]
[    0.000000] Primary instruction cache 64kB, VIPT, 4-way, linesize 32 bytes.
[    0.000000] Primary data cache 32kB, 4-way, VIPT, cache aliases, linesize 32 bytes
[    0.000000] Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 16256
[    0.000000] Kernel command line:  board=CUS531 console=ttyS0,115200 mtdparts=spi0.0:256k(u-boot)ro,64k(u-boot-env),6464k(rootfs1),6464k(rootfs),64k(apl),64k(config),1472k(kernel1),1472k(kernel2),64k(art) rootfstype=squashfs,jffs2 noinitrd mem=64M rootfstype=squashfs,jffs2 noinitrd
[    0.000000] PID hash table entries: 256 (order: -2, 1024 bytes)
[    0.000000] Dentry cache hash table entries: 8192 (order: 3, 32768 bytes)
[    0.000000] Inode-cache hash table entries: 4096 (order: 2, 16384 bytes)
[    0.000000] Writing ErrCtl register=00000000
[    0.000000] Readback ErrCtl register=00000000
[    0.000000] Memory: 60692K/65536K available (2725K kernel code, 127K rwdata, 580K rodata, 208K init, 185K bss, 4844K reserved)
[    0.000000] SLUB: HWalign=32, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
[    0.000000] NR_IRQS:51
[    0.000000] Clocks: CPU:650.000MHz, DDR:597.167MHz, AHB:216.666MHz, Ref:25.000MHz
[    0.000000] Calibrating delay loop... 432.53 BogoMIPS (lpj=2162688)
[    0.060000] pid_max: default: 32768 minimum: 301
[    0.060000] Mount-cache hash table entries: 1024 (order: 0, 4096 bytes)
[    0.070000] Mountpoint-cache hash table entries: 1024 (order: 0, 4096 bytes)
[    0.070000] NET: Registered protocol family 16
[    0.080000] MIPS: machine is Qualcomm Atheros CUS531 reference board
[    0.090000] ar71xx: invalid MDIO id 1
[    0.500000] bio: create slab <bio-0> at 0
[    0.510000] Switched to clocksource MIPS
[    0.510000] NET: Registered protocol family 2
[    0.520000] TCP established hash table entries: 1024 (order: 0, 4096 bytes)
[    0.520000] TCP bind hash table entries: 1024 (order: 0, 4096 bytes)
[    0.530000] TCP: Hash tables configured (established 1024 bind 1024)
[    0.540000] TCP: reno registered
[    0.540000] UDP hash table entries: 256 (order: 0, 4096 bytes)
[    0.540000] UDP-Lite hash table entries: 256 (order: 0, 4096 bytes)
[    0.550000] NET: Registered protocol family 1
[    0.560000] futex hash table entries: 256 (order: -1, 3072 bytes)
[    0.570000] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[    0.580000] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
[    0.590000] msgmni has been set to 118
[    0.600000] io scheduler noop registered
[    0.600000] io scheduler deadline registered (default)
[    0.610000] Serial: 8250/16550 driver, 1 ports, IRQ sharing disabled
[    0.630000] serial8250.0: ttyS0 at MMIO 0x18020000 (irq = 11, base_baud = 1562500) is a 16550A
[    0.640000] console [ttyS0] enabled
[    0.640000] console [ttyS0] enabled
[    0.650000] bootconsole [early0] disabled
[    0.650000] bootconsole [early0] disabled
[    0.660000] m25p80 spi0.0: found gd25q128, expected m25p80
[    0.660000] m25p80 spi0.0: gd25q128 (16384 Kbytes)
[    0.670000] 9 cmdlinepart partitions found on MTD device spi0.0
[    0.680000] Creating 9 MTD partitions on "spi0.0":
[    0.680000] 0x000000000000-0x000000040000 : "u-boot"
[    0.690000] 0x000000040000-0x000000050000 : "u-boot-env"
[    0.690000] 0x000000050000-0x0000006a0000 : "rootfs1"
[    0.700000] 0x0000006a0000-0x000000cf0000 : "rootfs"
[    0.710000] mtd: device 3 (rootfs) set to be root filesystem
[    0.710000] 1 squashfs-split partitions found on MTD device rootfs
[    0.720000] 0x000000c60000-0x000000cf0000 : "rootfs_data"
[    0.730000] 0x000000cf0000-0x000000d00000 : "apl"
[    0.730000] 0x000000d00000-0x000000d10000 : "config"
[    0.740000] 0x000000d10000-0x000000e80000 : "kernel1"
[    0.750000] 0x000000e80000-0x000000ff0000 : "kernel2"
[    0.750000] 0x000000ff0000-0x000001000000 : "art"
[    0.780000] libphy: ag71xx_mdio: probed
[    1.420000] ag71xx ag71xx.0: connected to PHY at ag71xx-mdio.1:04 [uid=004dd042, driver=Generic PHY]
[    1.430000] eth0: Atheros AG71xx at 0xb9000000, irq 4, mode:MII
[    1.990000] ag71xx-mdio.1: Found an AR934X built-in switch
[    3.020000] eth1: Atheros AG71xx at 0xba000000, irq 5, mode:GMII
[    3.020000] TCP: cubic registered
[    3.030000] NET: Registered protocol family 17
[    3.030000] 8021q: 802.1Q VLAN Support v1.8
[    3.040000] VFS: Mounted root (squashfs filesystem) readonly on device 31:3.
[    3.050000] Freeing unused kernel memory: 208K (803bc000 - 803f0000)
procd: Console is alive
procd: - watchdog -
procd: - preinit -
[    4.540000] random: mktemp urandom read with 40 bits of entropy available
Press the [f] key and hit [enter] to enter failsafe mode
Press the [1], [2], [3] or [4] key and hit [enter] to select the debug level
[    5.570000] usbcore: registered new interface driver usbfs
[    5.580000] usbcore: registered new interface driver hub
[    5.580000] usbcore: registered new device driver usb
[    5.600000] Button Hotplug driver version 0.4.1
[    5.620000] SCSI subsystem initialized
[    5.630000] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[    5.630000] ehci-platform: EHCI generic platform driver
[    5.640000] ehci-platform ehci-platform: EHCI Host Controller
[    5.650000] ehci-platform ehci-platform: new USB bus registered, assigned bus number 1
[    5.660000] ehci-platform ehci-platform: irq 3, io mem 0x1b000000
[    5.680000] ehci-platform ehci-platform: USB 2.0 started, EHCI 1.00
[    5.680000] hub 1-0:1.0: USB hub found
[    5.690000] hub 1-0:1.0: 1 port detected
[    5.700000] usbcore: registered new interface driver usb-storage
[    6.010000] usb 1-1: new high-speed USB device number 2 using ehci-platform
[    6.160000] hub 1-1:1.0: USB hub found
[    6.160000] hub 1-1:1.0: 4 ports detected
[    6.450000] usb 1-1.2: new full-speed USB device number 3 using ehci-platform
jffs2 is ready
[    7.790000] jffs2: notice: (342) jffs2_build_xattr_subsystem: complete building xattr subsystem, 1 of xdatum (0 unchecked, 0 orphan) and 247 of xref (0 dead, 228 orphan) found.
switching to jffs2
procd: - early -
procd: - watchdog -
procd: - ubus -
procd: - init -
Please press Enter to activate this console.
[    9.630000] NET: Registered protocol family 10
[    9.650000] nf_conntrack version 0.5.0 (951 buckets, 3804 max)
[    9.670000] ip6_tables: (C) 2000-2006 Netfilter Core Team
[    9.690000] Loading modules backported from Linux version v3.2-rc1-229534-gb3e71d7
[    9.700000] Backport generated by backports.git backports-20150129-0-gdd4a670
[    9.710000] ip_tables: (C) 2000-2006 Netfilter Core Team
[    9.730000] nf_conntrack_rtsp v0.6.21 loading
[    9.750000] nf_nat_rtsp v0.6.21 loading
[    9.770000] usbcore: registered new interface driver usbserial
[    9.780000] usbcore: registered new interface driver usbserial_generic
[    9.780000] usbserial: USB Serial support registered for generic
[    9.810000] xt_time: kernel timezone is -0000
[    9.850000] cfg80211: Calling CRDA to update world regulatory domain
[    9.850000] cfg80211: World regulatory domain updated:
[    9.860000] cfg80211:  DFS Master region: unset
[    9.860000] cfg80211:   (start_freq - end_freq @ bandwidth), (max_antenna_gain, max_eirp), (dfs_cac_time)
[    9.870000] cfg80211:   (2402000 KHz - 2472000 KHz @ 40000 KHz), (N/A, 2000 mBm), (N/A)
[    9.880000] cfg80211:   (2457000 KHz - 2482000 KHz @ 40000 KHz), (N/A, 2000 mBm), (N/A)
[    9.890000] cfg80211:   (2474000 KHz - 2494000 KHz @ 20000 KHz), (N/A, 2000 mBm), (N/A)
[    9.900000] cfg80211:   (5170000 KHz - 5250000 KHz @ 80000 KHz, 160000 KHz AUTO), (N/A, 2000 mBm), (N/A)
[    9.910000] cfg80211:   (5250000 KHz - 5330000 KHz @ 80000 KHz, 160000 KHz AUTO), (N/A, 2000 mBm), (0 s)
[    9.920000] cfg80211:   (5490000 KHz - 5730000 KHz @ 160000 KHz), (N/A, 2000 mBm), (0 s)
[    9.930000] cfg80211:   (5735000 KHz - 5835000 KHz @ 80000 KHz), (N/A, 2000 mBm), (N/A)
[    9.930000] cfg80211:   (57240000 KHz - 63720000 KHz @ 2160000 KHz), (N/A, 0 mBm), (N/A)
[    9.940000] usbcore: registered new interface driver cp210x
[    9.950000] usbserial: USB Serial support registered for cp210x
[    9.970000] usbcore: registered new interface driver ftdi_sio
[    9.970000] usbserial: USB Serial support registered for FTDI USB Serial Device
[   10.100000] usbcore: registered new interface driver pl2303
[   10.100000] usbserial: USB Serial support registered for pl2303
[   10.110000] pl2303 1-1.2:1.0: pl2303 converter detected
[   10.120000] usb 1-1.2: pl2303 converter now attached to ttyUSB0
[   10.140000] PPP generic driver version 2.4.2
[   10.150000] NET: Registered protocol family 24
[   10.290000] ieee80211 phy0: Atheros AR9531 Rev:2 mem=0xb8100000, irq=47
[   10.300000] cfg80211: Calling CRDA for country: DE
[   10.310000] cfg80211: Regulatory domain changed to country: DE
[   10.310000] cfg80211:  DFS Master region: ETSI
[   10.320000] cfg80211:   (start_freq - end_freq @ bandwidth), (max_antenna_gain, max_eirp), (dfs_cac_time)
[   10.330000] cfg80211:   (2400000 KHz - 2483000 KHz @ 40000 KHz), (N/A, 2000 mBm), (N/A)
[   10.330000] cfg80211:   (5150000 KHz - 5250000 KHz @ 80000 KHz, 200000 KHz AUTO), (N/A, 2000 mBm), (N/A)
[   10.340000] cfg80211:   (5250000 KHz - 5350000 KHz @ 80000 KHz, 200000 KHz AUTO), (N/A, 2000 mBm), (0 s)
[   10.350000] cfg80211:   (5470000 KHz - 5590000 KHz @ 80000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (0 s)
[   10.360000] cfg80211:   (5590000 KHz - 5650000 KHz @ 40000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (600 s)
[   10.370000] cfg80211:   (5650000 KHz - 5725000 KHz @ 40000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (0 s)
[   10.380000] cfg80211:   (57000000 KHz - 66000000 KHz @ 2160000 KHz), (N/A, 4000 mBm), (N/A)
START WATCHDOG   =====
START GoHeishaMon APL=====
[   19.360000] random: nonblocking pool is initialized
[   21.940000] IPv6: ADDRCONF(NETDEV_UP): eth1: link is not ready
[   21.950000] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
try to update configfile: /etc/gh/configPS Watch=====
[   22.910000] usb 1-1.1: new high-speed USB device number 4 using ehci-platform
[   23.030000] usb-storage 1-1.1:1.0: USB Mass Storage device detected
[   23.030000] scsi0 : usb-storage 1-1.1:1.0
[   23.060000] IPv6: ADDRCONF(NETDEV_UP): wlan0: link is not ready
procd: - init complete -
[   24.140000] scsi 0:0:0:0: Direct-Access     IS917    innostor         1.00 PQ: 0 ANSI: 6
[   24.180000] sd 0:0:0:0: [sda] 15728640 512-byte logical blocks: (8.05 GB/7.50 GiB)
[   24.200000] sd 0:0:0:0: [sda] Write Protect is off
[   24.200000] sd 0:0:0:0: [sda] Write cache: disabled, read cache: disabled, doesn't support DPO or FUA
[   24.270000]  sda:
[   24.290000] sd 0:0:0:0: [sda] Attached SCSI removable disk
[   27.110000] wlan0: authenticate with 70:3a:cb:7d:7c:95
[   27.130000] wlan0: send auth to 70:3a:cb:7d:7c:95 (try 1/3)
[   27.140000] wlan0: authenticated
[   27.160000] wlan0: associate with 70:3a:cb:7d:7c:95 (try 1/3)
[   27.170000] wlan0: RX AssocResp from 70:3a:cb:7d:7c:95 (capab=0x1031 status=0 aid=7)
[   27.180000] wlan0: associated
[   27.180000] IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
[   27.220000] cfg80211: Calling CRDA for country: DE
[   27.230000] cfg80211: Regulatory domain changed to country: DE
[   27.230000] cfg80211:  DFS Master region: ETSI
[   27.240000] cfg80211:   (start_freq - end_freq @ bandwidth), (max_antenna_gain, max_eirp), (dfs_cac_time)
[   27.250000] cfg80211:   (2400000 KHz - 2483000 KHz @ 40000 KHz), (N/A, 2000 mBm), (N/A)
[   27.250000] cfg80211:   (5150000 KHz - 5250000 KHz @ 80000 KHz, 200000 KHz AUTO), (N/A, 2000 mBm), (N/A)
[   27.260000] cfg80211:   (5250000 KHz - 5350000 KHz @ 80000 KHz, 200000 KHz AUTO), (N/A, 2000 mBm), (0 s)
[   27.270000] cfg80211:   (5470000 KHz - 5590000 KHz @ 80000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (0 s)
[   27.280000] cfg80211:   (5590000 KHz - 5650000 KHz @ 40000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (600 s)
[   27.290000] cfg80211:   (5650000 KHz - 5725000 KHz @ 40000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (0 s)
[   27.300000] cfg80211:   (57000000 KHz - 66000000 KHz @ 2160000 KHz), (N/A, 4000 mBm), (N/A)
[   27.320000] wlan0: Limiting TX power to 20 (20 - 0) dBm as advertised by 70:3a:cb:7d:7c:95

Can you see some problem here?

**Comment by MiG-41** [comment]
No ,here everything is ok. GoHeishamon also starts.
Also config file looks like it is being copied.

**Comment by stefanbbb** [comment]
But log stops after initialising wlan. What would be the next action in startup, which is not done or hanging or whatever? Is it mqtt?

**Comment by MiG-41** [comment]
GoHeishamon cousing stopping log .(precisly is changing console speed  to 9600 ). 
You can change The Speed to 9600 ,Then The log wilk move further. 
Like i said please Press f and  enter jest when it ask. Then do uci show ,and after that Firstboot ,confirm it ,and at The end reboot.
After next reboot ssh should be availble, and it will be easier from there see what GoHeishamon is saying.

**Comment by stefanbbb** [comment]
OK, got it!
Here is the output of uci show
root@(none):/# uci show
dhcp6c.basic=dhcp6c
dhcp6c.basic.enabled=0
dhcp6c.basic.interface=wan
dhcp6c.basic.dns=dnsmasq
dhcp6c.basic.debug=0
dhcp6c.basic.pd=1
dhcp6c.basic.na=0
dhcp6c.basic.rapid_commit=1
dhcp6c.basic.domain_name_servers=1
dhcp6c.basic.domain_name=0
dhcp6c.basic.ntp_servers=0
dhcp6c.basic.sip_server_address=0
dhcp6c.basic.sip_server_domain_name=0
dhcp6c.basic.nis_server_address=0
dhcp6c.basic.nis_domain_name=0
dhcp6c.basic.nisp_server_address=0
dhcp6c.basic.nisp_domain_name=0
dhcp6c.basic.bcmcs_server_address=0
dhcp6c.basic.bcmcs_server_domain_name=0
dhcp6c.basic.script=/usr/bin/dhcp6c-state
dhcp6c.loopback=interface
dhcp6c.loopback.enabled=1
dhcp6c.loopback.sla_id=0
dhcp6c.loopback.sla_len=4
dhcp6c.lan=interface
dhcp6c.lan.enabled=1
dhcp6c.lan.sla_id=1
dhcp6c.lan.sla_len=4
dropbear.@dropbear[0]=dropbear
dropbear.@dropbear[0].PasswordAuth=on
dropbear.@dropbear[0].RootPasswordAuth=on
dropbear.@dropbear[0].Port=22
firewall.@defaults[0]=defaults
firewall.@defaults[0].syn_flood=1
firewall.@defaults[0].input=ACCEPT
firewall.@defaults[0].output=ACCEPT
firewall.@defaults[0].forward=REJECT
firewall.@zone[0]=zone
firewall.@zone[0].name=lan
firewall.@zone[0].network=lan
firewall.@zone[0].input=ACCEPT
firewall.@zone[0].output=ACCEPT
firewall.@zone[0].forward=ACCEPT
firewall.@zone[1]=zone
firewall.@zone[1].name=wan
firewall.@zone[1].network=wan wan6
firewall.@zone[1].input=REJECT
firewall.@zone[1].output=ACCEPT
firewall.@zone[1].forward=REJECT
firewall.@zone[1].masq=1
firewall.@zone[1].mtu_fix=1
firewall.@forwarding[0]=forwarding
firewall.@forwarding[0].src=lan
firewall.@forwarding[0].dest=wan
firewall.@rule[0]=rule
firewall.@rule[0].name=Allow-DHCP-Renew
firewall.@rule[0].src=wan
firewall.@rule[0].proto=udp
firewall.@rule[0].dest_port=68
firewall.@rule[0].target=ACCEPT
firewall.@rule[0].family=ipv4
firewall.@rule[1]=rule
firewall.@rule[1].name=Allow-Ping
firewall.@rule[1].src=wan
firewall.@rule[1].proto=icmp
firewall.@rule[1].icmp_type=echo-request
firewall.@rule[1].family=ipv4
firewall.@rule[1].target=ACCEPT
firewall.@rule[2]=rule
firewall.@rule[2].name=Allow-DHCPv6
firewall.@rule[2].src=wan
firewall.@rule[2].proto=udp
firewall.@rule[2].src_ip=fe80::/10
firewall.@rule[2].src_port=547
firewall.@rule[2].dest_ip=fe80::/10
firewall.@rule[2].dest_port=546
firewall.@rule[2].family=ipv6
firewall.@rule[2].target=ACCEPT
firewall.@rule[3]=rule
firewall.@rule[3].name=Allow-ICMPv6-Input
firewall.@rule[3].src=wan
firewall.@rule[3].proto=icmp
firewall.@rule[3].icmp_type=echo-request echo-reply destination-unreachable packet-too-big time-exceeded bad-header unknown-header-type router-solicitation neighbour-solicitation router-advertisement neighbour-advertisement
firewall.@rule[3].limit=1000/sec
firewall.@rule[3].family=ipv6
firewall.@rule[3].target=ACCEPT
firewall.@rule[4]=rule
firewall.@rule[4].name=Allow-ICMPv6-Forward
firewall.@rule[4].src=wan
firewall.@rule[4].dest=*
firewall.@rule[4].proto=icmp
firewall.@rule[4].icmp_type=echo-request echo-reply destination-unreachable packet-too-big time-exceeded bad-header unknown-header-type
firewall.@rule[4].limit=1000/sec
firewall.@rule[4].family=ipv6
firewall.@rule[4].target=ACCEPT
firewall.@include[0]=include
firewall.@include[0].path=/etc/firewall.user
luci.main=core
luci.main.lang=auto
luci.main.mediaurlbase=/luci-static/openwrt.org
luci.main.resourcebase=/luci-static/resources
luci.flash_keep=extern
luci.flash_keep.uci=/etc/config/
luci.flash_keep.dropbear=/etc/dropbear/
luci.flash_keep.openvpn=/etc/openvpn/
luci.flash_keep.passwd=/etc/passwd
luci.flash_keep.opkg=/etc/opkg.conf
luci.flash_keep.firewall=/etc/firewall.user
luci.flash_keep.uploads=/lib/uci/upload/
luci.languages=internal
luci.sauth=internal
luci.sauth.sessionpath=/tmp/luci-sessions
luci.sauth.sessiontime=3600
luci.ccache=internal
luci.ccache.enable=1
luci.themes=internal
mcproxy.config=mcproxy
mcproxy.config.protocol=IGMPv3
mcproxy.@pinstance[0]=pinstance
mcproxy.@pinstance[0].name=mcproxy1
system.@system[0]=system
system.@system[0].hostname=OpenWrt
system.@system[0].timezone=UTC
system.ntp=timeserver
system.ntp.server=0.openwrt.pool.ntp.org 1.openwrt.pool.ntp.org 2.openwrt.pool.ntp.org 3.openwrt.pool.ntp.org
system.ntp.enable_server=0
ucitrack.@network[0]=network
ucitrack.@network[0].init=network
ucitrack.@network[0].affects=dhcp radvd
ucitrack.@wireless[0]=wireless
ucitrack.@wireless[0].affects=network
ucitrack.@firewall[0]=firewall
ucitrack.@firewall[0].init=firewall
ucitrack.@firewall[0].affects=luci-splash qos miniupnpd
ucitrack.@olsr[0]=olsr
ucitrack.@olsr[0].init=olsrd
ucitrack.@dhcp[0]=dhcp
ucitrack.@dhcp[0].init=dnsmasq
ucitrack.@dropbear[0]=dropbear
ucitrack.@dropbear[0].init=dropbear
ucitrack.@httpd[0]=httpd
ucitrack.@httpd[0].init=httpd
ucitrack.@fstab[0]=fstab
ucitrack.@fstab[0].init=fstab
ucitrack.@qos[0]=qos
ucitrack.@qos[0].init=qos
ucitrack.@system[0]=system
ucitrack.@system[0].init=led
ucitrack.@system[0].affects=luci_statistics
ucitrack.@luci_splash[0]=luci_splash
ucitrack.@luci_splash[0].init=luci_splash
ucitrack.@upnpd[0]=upnpd
ucitrack.@upnpd[0].init=miniupnpd
ucitrack.@ntpclient[0]=ntpclient
ucitrack.@ntpclient[0].init=ntpclient
ucitrack.@samba[0]=samba
ucitrack.@samba[0].init=samba
ucitrack.@tinyproxy[0]=tinyproxy
ucitrack.@tinyproxy[0].init=tinyproxy
ucitrack.@6relayd[0]=6relayd
ucitrack.@6relayd[0].init=6relayd
uhttpd.main=uhttpd
uhttpd.main.listen_http=0.0.0.0:80
uhttpd.main.listen_https=0.0.0.0:443
uhttpd.main.home=/www
uhttpd.main.rfc1918_filter=1
uhttpd.main.max_requests=3
uhttpd.main.max_connections=100
uhttpd.main.cert=/etc/uhttpd.crt
uhttpd.main.key=/etc/uhttpd.key
uhttpd.main.cgi_prefix=/cgi-bin
uhttpd.main.script_timeout=60
uhttpd.main.network_timeout=30
uhttpd.main.http_keepalive=20
uhttpd.main.tcp_keepalive=1
uhttpd.px5g=cert
uhttpd.px5g.days=730
uhttpd.px5g.bits=1024
uhttpd.px5g.country=DE
uhttpd.px5g.state=Berlin
uhttpd.px5g.location=Berlin
uhttpd.px5g.commonname=OpenWrt
upnpd.config=upnpd
upnpd.config.enable_natpmp=1
upnpd.config.enable_upnp=1
upnpd.config.secure_mode=1
upnpd.config.log_output=0
upnpd.config.download=1024
upnpd.config.upload=512
upnpd.config.external_iface=wan
upnpd.config.internal_iface=lan
upnpd.config.port=5000
upnpd.config.upnp_lease_file=/var/upnp.leases
upnpd.@perm_rule[0]=perm_rule
upnpd.@perm_rule[0].action=allow
upnpd.@perm_rule[0].ext_ports=1024-65535
upnpd.@perm_rule[0].int_addr=0.0.0.0/0
upnpd.@perm_rule[0].int_ports=1024-65535
upnpd.@perm_rule[0].comment=Allow high ports
upnpd.@perm_rule[1]=perm_rule
upnpd.@perm_rule[1].action=deny
upnpd.@perm_rule[1].ext_ports=0-65535
upnpd.@perm_rule[1].int_addr=0.0.0.0/0
upnpd.@perm_rule[1].int_ports=0-65535
upnpd.@perm_rule[1].comment=Default deny
root@(none):/# Firstboot
ash: Firstboot: not found
root@(none):/# firstboot
This will erase all settings and remove any installed packages. Are you sure? [N/y]
y
/dev/mtdblock4 is not mounted, erasing it
erasing 0 10000
erasing 10000 10000
erasing 20000 10000
erasing 30000 10000
erasing 40000 10000
erasing 50000 10000
erasing 60000 10000
erasing 70000 10000
erasing 80000 10000
root@(none):/#

U-Boot 1.1.4-g4d980152-dirty (Jun 24 2016 - 10:12:23)

But after that, I have no wlan-connection at all (no ping). So, also no ssh;-(

**Comment by stefanbbb** [comment]
btw: currently I have nice stable white led and somehow flickering link led (but still no mqtt)

**Comment by MiG-41** [comment]
Sorry,forgot to mentioned here ,that firstboot also will  boeing back wifi to default, so now only cable  Ethernet connection  availble.
Same with GoHeishamon,There are default config.

**Comment by stefanbbb** [comment]
but how can I set up wlan new? When I remember, in original SW I had to place an usb drive with a config file too. Is it the same way with GoHeishamon?

**Comment by MiG-41** [comment]
No it will be different ( similar to OpenWRT routers). 
Did you removed some lines from uci show results ? I miss wireless settings....
After firstboot and reset are you able to connect it via ethernet and see if ssh and luci is availble ?

**Comment by stefanbbb** [comment]
ssh is there, but I need user/pw

**Comment by MiG-41** [comment]
root / GoHeishaMonpass

**Comment by stefanbbb** [comment]
I'm in :-)
What I have to check?

**Comment by stefanbbb** [comment]
The former post was the complete output of uci show.
The log of the complete new boot after firstboot is here:
MTD device rootfs
[    0.720000] 0x000000c60000-0x000000cf0000 : "rootfs_data"
[    0.730000] 0x000000cf0000-0x000000d00000 : "apl"
[    0.730000] 0x000000d00000-0x000000d10000 : "config"
[    0.740000] 0x000000d10000-0x000000e80000 : "kernel1"
[    0.750000] 0x000000e80000-0x000000ff0000 : "kernel2"
[    0.750000] 0x000000ff0000-0x000001000000 : "art"
[    0.780000] libphy: ag71xx_mdio: probed
[    1.420000] ag71xx ag71xx.0: connected to PHY at ag71xx-mdio.1:04 [uid=004dd042, driver=Generic PHY]
[    1.430000] eth0: Atheros AG71xx at 0xb9000000, irq 4, mode:MII
[    1.990000] ag71xx-mdio.1: Found an AR934X built-in switch
[    3.020000] eth1: Atheros AG71xx at 0xba000000, irq 5, mode:GMII
[    3.020000] TCP: cubic registered
[    3.030000] NET: Registered protocol family 17
[    3.030000] 8021q: 802.1Q VLAN Support v1.8
[    3.040000] VFS: Mounted root (squashfs filesystem) readonly on device 31:3.
[    3.050000] Freeing unused kernel memory: 208K (803bc000 - 803f0000)
procd: Console is alive
procd: - watchdog -
procd: - preinit -
[    4.550000] random: mktemp urandom read with 41 bits of entropy available
Press the [f] key and hit [enter] to enter failsafe mode
Press the [1], [2], [3] or [4] key and hit [enter] to select the debug level
[    5.570000] usbcore: registered new interface driver usbfs
[    5.580000] usbcore: registered new interface driver hub
[    5.580000] usbcore: registered new device driver usb
[    5.590000] Button Hotplug driver version 0.4.1
[    5.610000] SCSI subsystem initialized
[    5.620000] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[    5.620000] ehci-platform: EHCI generic platform driver
[    5.630000] ehci-platform ehci-platform: EHCI Host Controller
[    5.640000] ehci-platform ehci-platform: new USB bus registered, assigned bus number 1
[    5.650000] ehci-platform ehci-platform: irq 3, io mem 0x1b000000
[    5.670000] ehci-platform ehci-platform: USB 2.0 started, EHCI 1.00
[    5.670000] hub 1-0:1.0: USB hub found
[    5.680000] hub 1-0:1.0: 1 port detected
[    5.690000] usbcore: registered new interface driver usb-storage
[    6.000000] usb 1-1: new high-speed USB device number 2 using ehci-platform
[    6.150000] hub 1-1:1.0: USB hub found
[    6.150000] hub 1-1:1.0: 4 ports detected
[    6.440000] usb 1-1.2: new full-speed USB device number 3 using ehci-platform
No jffs2 marker was found
procd: - early -
procd: - watchdog -
procd: - ubus -
procd: - init -
Please press Enter to activate this console.
fwupdate. no saved uci config. do nothing
[    9.650000] NET: Registered protocol family 10
[    9.670000] nf_conntrack version 0.5.0 (951 buckets, 3804 max)
[    9.690000] ip6_tables: (C) 2000-2006 Netfilter Core Team
[    9.710000] Loading modules backported from Linux version v3.2-rc1-229534-gb3e71d7
[    9.710000] Backport generated by backports.git backports-20150129-0-gdd4a670
[    9.720000] ip_tables: (C) 2000-2006 Netfilter Core Team
[    9.750000] nf_conntrack_rtsp v0.6.21 loading
[    9.770000] nf_nat_rtsp v0.6.21 loading
[    9.790000] usbcore: registered new interface driver usbserial
[    9.790000] usbcore: registered new interface driver usbserial_generic
[    9.800000] usbserial: USB Serial support registered for generic
[    9.830000] xt_time: kernel timezone is -0000
[    9.870000] cfg80211: Calling CRDA to update world regulatory domain
[    9.870000] cfg80211: World regulatory domain updated:
[    9.880000] cfg80211:  DFS Master region: unset
[    9.880000] cfg80211:   (start_freq - end_freq @ bandwidth), (max_antenna_gain, max_eirp), (dfs_cac_time)
[    9.890000] cfg80211:   (2402000 KHz - 2472000 KHz @ 40000 KHz), (N/A, 2000 mBm), (N/A)
[    9.900000] cfg80211:   (2457000 KHz - 2482000 KHz @ 40000 KHz), (N/A, 2000 mBm), (N/A)
[    9.910000] cfg80211:   (2474000 KHz - 2494000 KHz @ 20000 KHz), (N/A, 2000 mBm), (N/A)
[    9.920000] cfg80211:   (5170000 KHz - 5250000 KHz @ 80000 KHz, 160000 KHz AUTO), (N/A, 2000 mBm), (N/A)
[    9.930000] cfg80211:   (5250000 KHz - 5330000 KHz @ 80000 KHz, 160000 KHz AUTO), (N/A, 2000 mBm), (0 s)
[    9.940000] cfg80211:   (5490000 KHz - 5730000 KHz @ 160000 KHz), (N/A, 2000 mBm), (0 s)
[    9.950000] cfg80211:   (5735000 KHz - 5835000 KHz @ 80000 KHz), (N/A, 2000 mBm), (N/A)
[    9.950000] cfg80211:   (57240000 KHz - 63720000 KHz @ 2160000 KHz), (N/A, 0 mBm), (N/A)
[    9.960000] usbcore: registered new interface driver cp210x
[    9.980000] usbserial: USB Serial support registered for cp210x
[    9.980000] usbcore: registered new interface driver ftdi_sio
[   10.000000] usbserial: USB Serial support registered for FTDI USB Serial Device
[   10.120000] usbcore: registered new interface driver pl2303
[   10.120000] usbserial: USB Serial support registered for pl2303
[   10.130000] pl2303 1-1.2:1.0: pl2303 converter detected
[   10.150000] usb 1-1.2: pl2303 converter now attached to ttyUSB0
[   10.160000] PPP generic driver version 2.4.2
[   10.170000] NET: Registered protocol family 24
[   10.310000] ieee80211 phy0: Atheros AR9531 Rev:2 mem=0xb8100000, irq=47
[   10.320000] cfg80211: Calling CRDA for country: DE
[   10.330000] cfg80211: Regulatory domain changed to country: DE
[   10.330000] cfg80211:  DFS Master region: ETSI
[   10.340000] cfg80211:   (start_freq - end_freq @ bandwidth), (max_antenna_gain, max_eirp), (dfs_cac_time)
[   10.350000] cfg80211:   (2400000 KHz - 2483000 KHz @ 40000 KHz), (N/A, 2000 mBm), (N/A)
[   10.350000] cfg80211:   (5150000 KHz - 5250000 KHz @ 80000 KHz, 200000 KHz AUTO), (N/A, 2000 mBm), (N/A)
[   10.360000] cfg80211:   (5250000 KHz - 5350000 KHz @ 80000 KHz, 200000 KHz AUTO), (N/A, 2000 mBm), (0 s)
[   10.370000] cfg80211:   (5470000 KHz - 5590000 KHz @ 80000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (0 s)
[   10.380000] cfg80211:   (5590000 KHz - 5650000 KHz @ 40000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (600 s)
[   10.390000] cfg80211:   (5650000 KHz - 5725000 KHz @ 40000 KHz, 255000 KHz AUTO), (N/A, 2700 mBm), (0 s)
[   10.400000] cfg80211:   (57000000 KHz - 66000000 KHz @ 2160000 KHz), (N/A, 4000 mBm), (N/A)
[   19.310000] random: nonblocking pool is initialized
[   20.140000] jffs2: notice: (997) jffs2_build_xattr_subsystem: complete building xattr subsystem, 0 of xdatum (0 unchecked, 0 orphan) and 0 of xref (0 dead, 0 orphan) found.
START WATCHDOG   =====
START GoHeishaMon APL=====
[   22.720000] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
[   22.720000] device eth0 entered promiscuous mode
[   22.770000] IPv6: ADDRCONF(NETDEV_UP): br-lan: link is not ready
[   23.800000] IPv6: ADDRCONF(NETDEV_UP): eth1: link is not ready
PS Watch=====
try to update configfile: /etc/gh/configmap[gpio-0:lo gpio-1:lo gpio-10:hi gpio-13:lo gpio-15:lo gpio-16:hi gpio-2:lo gpio-3:lo]

**Comment by MiG-41** [comment]
so if you in ,first do :
killall -9 GoHeishaMon_MIPSUPX
and then run it again /usr/bin/GoHeishaMon_MIPSUPX
and look what is wrong.

**Comment by stefanbbb** [comment]
in ssh, I find with uci show also wifi settings:
wireless.radio0=wifi-device
wireless.radio0.type=mac80211
wireless.radio0.channel=11
wireless.radio0.hwmode=11ng
wireless.radio0.path=platform/qca953x_wmac
wireless.radio0.ht_capab=LDPC SHORT-GI-20 SHORT-GI-40 TX-STBC RX-STBC1 DSSS_CCK-40
wireless.radio0.htmode=HT20
wireless.radio0.disabled=1
wireless.@wifi-iface[0]=wifi-iface
wireless.@wifi-iface[0].device=radio0
wireless.@wifi-iface[0].network=lan
wireless.@wifi-iface[0].mode=ap
wireless.@wifi-iface[0].ssid=OpenWrt
wireless.@wifi-iface[0].encryption=none

**Comment by MiG-41** [comment]
yes , that is a mistery why it was not before...

**Comment by stefanbbb** [comment]
Done, now I get a lot of hex dumps:

Checksum and header received ok!
Total reads : 84.000000 and total good reads : 82.000000 (97.62 %)
Fail to publish, not Connectedread ma status true
map[gpio-0:lo gpio-1:lo gpio-10:hi gpio-11:lo gpio-13:hi gpio-15:hi gpio-16:hi gpio-2:hi gpio-3:lo]
map[gpio-0:lo gpio-1:lo gpio-10:hi gpio-11:lo gpio-13:hi gpio-15:hi gpio-16:hi gpio-2:hi gpio-3:lo]
Fail to connect broker, network Error : dial tcp 127.0.0.1:1883: connect: connection refusedsent bytes: 110 with checksum: 18
71 6C 01 10 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
Received 203 bytes data

71 C8 01 10 55 95 53 49 00 55 00 00 00 00 00 00 00 00 00 00 19 15 11 55 15 9D 55 05 0A 00 00 00 00 00 00 00 00 00 99 8D A3 8A B4 71 71 5D 99 00 00 00 00 00 00 00 00 00 00 00 80 85 15 8A 85 85 D0 7B 78 1F 7E 1F 1F 79 79 8D 8D 99 A3 7B 8F A3 A3 7B 8F 98 85 80 8D 8A 94 9E 8A 8A 94 9E 81 8B 8A 11 3D 78 C1 0B 00 00 00 00 00 00 00 00 55 55 55 21 73 15 55 05 22 12 69 00 00 00 00 00 00 00 00 E2 CF 0B 15 33 42 D1 0B 18 33 93 00 00 93 93 93 32 32 8D 8A 32 32 32 99 99 93 92 95 93 93 61 94 61 97 42 01 01 01 00 00 22 00 01 01 01 01 79 79 01 01 9C 00 00 B6 01 00 01 00 00 01 00 00 0A 01 01 01 01 01 01 01 00 00 00 E3
Checksum and header received ok!
Total reads : 85.000000 and total good reads : 83.000000 (97.65 %)
Fail to publish, not Connectedread ma status true
^C

**Comment by MiG-41** [comment]
"Fail to connect broker, network Error : dial tcp 127.0.0.1:1883: connect: connection refused"
please insert a pendrive with config , and wait when config upload.
When it will happen probably you have to kill GoHeishaMon_MIPSUPX (first ctrl+z , and then by killall comand like in above post ,and start it)

**Comment by stefanbbb** [comment]
The pendrive with the config file named"GoHeishaMonConfig.new" was in all the time

**Comment by MiG-41** [comment]
Ok , strange. Anyway remove this pendrive ,ctrl-z  ,and killall -9 GoHeishaMon_MIPSUPX.
And if you know how to use vi editor , you can edit config file "vi /etc/gh/config"( or i.e. remontly edit this file via WinSCP)  and apply your settings  ,and then run /usr/bin/GoHeishaMon_MIPSUPX again , to see why it is not connecting to mqtt.

You can also do a "fwupdate sw" ,and reboot , to change the side to SmartCloud ,and install the newest 1.1.150 GoHeishaMon from scratch ( and following new readme.md)

**Comment by stefanbbb** [comment]
Great, done!
The config file was just filled with standard values, not with the values from the pen. I changed it manually and mqtt is running now.

Thank you very much for your great support!!:-) 

Now it would be just nice to know, how to set the wlan settings. And why the config file gets not updated. During startup, the LED of the pendrive was blinking. So, access to the file should be ok. Maybe some problem with the pen drive?

**Comment by MiG-41** [comment]
First , please update it to the newest 1.1.150 . I'm not sure why config file could not be updated , but it is modified on newer version , so this knowleadge will be useless anyway.
So go back to SmartCloud ( hope 3 buttons are working also ) , and install new one.

**Comment by stefanbbb** [comment]
Update is done, but now we have the same story again:
no mqtt, no ssh.
Are you sure that the config file has the right filename? (GoHeishaMonConfig.new)

**Comment by stefanbbb** [comment]
Can you provide a bin-file where the ssh is already activated?

**Comment by stefanbbb** [comment]
BTW: I edit the config file on a windows pc. I know that this sometimes leads to problems on Linux-based systems. Could that be the rootcause that the config file get not copied?

**Comment by stefanbbb** [comment]
After some reboots it's working now. mqtt is there, but when I try to activate ssh, I get an error in OSCommand out:
"umount: can't umount /overlay: Invalid argument
 - exit status 1"

**Comment by MiG-41** [comment]
Looks like you  performed command umount /overlay already and you trying to do it second time. Umount is posible only once :)
To be sure /overlay is unmounted , type command mount , if in answer there is line "/dev/mtdblock3 on /overlay type jffs2 (rw,noatime)"
it is still mounted , if not , it is already umounted.

But since you performed firstboot on your CZ-TAW1 already ,there is a chance that ssh will be already without doing it. Please wait 5-10min to build the ssh rsa2 key and try to connect.

**Comment by stefanbbb** [comment]
Ok, well done!
SSH is running.
I'm just trying to config wlan with openwrt doc, but I'm struggling a little bit. Can you give me the right parameters?

**Comment by MiG-41** [comment]
This is how orginally CZ-TAW1 is doing it:

uci set network.wlan=interface
uci set network.wlan.proto=dhcp
uci set wireless.radio0.channel=auto
uci set wireless.radio0.disabled=0
uci set wireless.@wifi-iface[0].network=wlan
uci set wireless.@wifi-iface[0].mode=sta
uci set wireless.@wifi-iface[0].encryption=psk2
uci set wireless.@wifi-iface[0].key=yourpassword
uci set wireless.@wifi-iface[0].ssid=yourSSID
uci commit
wifi

In this whay you will receive additional IP for WIFI ,so booth connections will be possible.

**Comment by stefanbbb** [comment]
Ooops, this sounds dangerous. My cable is not there anymore, and the soldering was a little bit fiddly.So, I want to be 100% sure that it works.
On openwrt.org I've read something like option mode 'sta'
        and option wds '1' (https://openwrt.org/docs/guide-user/network/wifi/atheroswds). Do we need that here, too?

**Comment by MiG-41** [comment]
Sta should be already set ( you can see current settings via uci show , or specyfy uci show wireless), but i added to above post this command. WDS is not a case ,you are not making a wireless bridge or reapeter,only client. Also WDS needs some implementation in drivers , i'm not sure if openwrt realese with CZ-TAW1 uses, have it.

Edited post above , checked ,and worked for me.

**Comment by stefanbbb** [comment]
Another question:
would it be possible to implement some kind of uboot boot script or similar in further versions? That would make handling with  ssh and network settings much more easier, I think. And we could avoid the fiddly handling during soldering these small cables. What do you think? Is that a possible improvement? It would help some stupid beginners like me a lot in setting up and administration. And if I make an error, I just would have to change the boot script.

**Comment by MiG-41** [comment]
For beginners only thing with is needed , to have always possibility to go back to SmartCloud , and have posiblility to upload the config file. 
First posibility now is "upgrated". Previously was , that GoHeishaMon procces took care about the handling of three buttons and it was a risk ,that if this proces stops ,also buttons will be not supported anymore. Now it is independent , and it is pretty sure ,that buttons will be always availble.
Second possiblility was always , but now is updated , that when the config is uploaded ,restart is performed and config is inplemented. 

SSH and luci is not needed for beginners. 

Wifi setup will be , if it was on SmartCloud side.

**Comment by lsochanowski** [comment]
guys i think this issue was closed and we can move on  :)

**Comment by stefanbbb** [comment]
OK, last comment:
This means, if ethernet connection is killed, I can now press the 3 buttons, configure wlan in original firmware and install GoHeishaMoon new by pressing the 3 buttons again, right?

**Comment by lsochanowski** [comment]
Yes ! For now it’s prefered Solution

**Comment by stefanbbb** [comment]
Just to let you know: wlan and lan are running with the new configuration!!!
Great! Thanks for supporting me so much!!
